packages/runtime-core/src/componentRenderContext.ts

```diff
@@ -89,10 +89,14 @@ export function withCtx(
       setBlockTracking(-1)
     }
     const prevInstance = setCurrentRenderingInstance(ctx)
-    const res = fn(...args)
-    setCurrentRenderingInstance(prevInstance)
-    if (renderFnWithContext._d) {
-      setBlockTracking(1)
+    let res
+    try {
```

Comment:
> IMO, if the `fn` throws an exception, the error message should be thrown to tell the user, rather than just ignoring it.

First, thank you for taking the time to review it. I've considered this before, but the error bubbles up into the render function and is caught and handled by it. Not sure if it's necessary to catch it more than once.

Reply:
Here it uses a `try` without `catch` should make the error bubble up instead of ignoring it.